src/test/java/org/apache/commons/lang3/time/DateUtilsTest.java

```diff
@@ -693,6 +693,59 @@ public void testToCalendar() {
             // expected
         }
     }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNotNull() {
+        assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date1, DateUtils.toCalendar(date1, zone).getTime());
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNull() {
+        try {
+            DateUtils.toCalendar(null, zone);
+            fail("Expected NullPointerException to be thrown when Date is null");
+        } catch(final NullPointerException npe) {
+            // expected
+        }
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithTimeZoneNotNull() {
+    	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the TimeZone back", zone, DateUtils.toCalendar(date1, zone).getTimeZone());
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithTimeZoneNull() {
+        try {
+            DateUtils.toCalendar(date1, null);
+            fail("Expected NullPointerException to be thrown when TimeZone is null");
+        } catch(final NullPointerException npe) {
+            // expected
+        }
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateAndTimeZoneNotNull() {
+    	Calendar c = DateUtils.toCalendar(date2, defaultZone);
+    	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date2, c.getTime());
+    	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the TimeZone back", defaultZone, c.getTimeZone());
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateAndTimeZoneNull() {
+    	try {
```

Comment:
you don't need a try-catch here. You can simply use `@Test(expected=NullPointerException.class)`